app/ide-desktop/lib/client/src/index.ts

```diff
@@ -74,6 +140,44 @@ class App {
         }
     }
 
+    processArguments() {
```

Comment:
1. I think this function name is misleading. It does not only processes arguments, but also tries to open the file.
2. I believe the line `let openedFile = ...` should be outside of this function, refactored to the module `client/src/file-assotiations`. It's result should be passed to this function instead.
3. `let` -> `const`. Please be sure you listed the file, as linter should catch it.

Reply:
> 1. I think this function name is misleading. It does not only processes arguments, but also tries to open the file.

As my understanding of `attemptingToOpenFile` goes (see above), the `processArguments` name is good. The `openedFile` is the misleading name, actually.

Reply:
Renamed `openedFile`.